.github/CODE_OF_CONDUCT.md

```diff
@@ -56,11 +56,11 @@ further defined and clarified by project maintainers.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported by contacting the project team at
-[scott.hardy@mindbrige.ai](mailto:scott.hardy@mindbrige.ai). All complaints will
-be reviewed and investigated and will result in a response that is deemed
-necessary and appropriate to the circumstances. The project team is obligated to
-maintain confidentiality with regard to the reporter of an incident. Further
-details of specific enforcement policies may be posted separately.
+[hardy613@gmail.com](mailto:hardy613@gmail.com). All complaints will be reviewed
+and investigated and will result in a response that is deemednecessary and
```

Comment:
some words gotsquishedtogether whith the change in line wrapping

Reply:
it'sdenoformattermessingwithme